projects/go-style-guide/src/app/features/ui-kit/components/off-canvas-docs/off-canvas-docs.component.ts

```diff
@@ -73,6 +73,18 @@ export class OffCanvasDocsComponent {
   </go-button>
   `;
 
+  largeOffCanvasExample: string = `
+  openOffCanvas() : void {
+    this.goOffCanvasService.openOffCanvas({
+      component: BasicTestComponent,
+      bindings: {
+        someBinding: 'monkey'
+      },
+      header: 'Test Header'
+    });
```

Comment:
I think maybe this is missing the `size` property 😬

Reply:
You may be looking at an old version of the file.